src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/validator/TransactionInputValidator.java

```diff
@@ -86,4 +87,15 @@ private void validateAmount(final TransactionDto transactionDto) {
 			return;
 		}
 	}
+
+	public void validatePendingStatus(final List<TransactionDto> transactionDtos) {
+		errors = new MapBindingResult(new HashMap<String, String>(), TransactionDto.class.getName());
+		final long pendingTransactionCount = transactionDtos.stream()
```

Comment:
It will be more ellegant with a negation of the Predicate
`		final Predicate<TransactionDto> isPendingStatus = i -> i.getTransactionStatus().equals(TransactionStatus.PENDING.getValue());
		if (transactionDtos.stream().filter(isPendingStatus.negate()).count() > 0) {`